.github/workflows/build_and_test.yml

```diff
@@ -339,6 +339,22 @@ jobs:
         env:
           TEST_RESULT_CONNSTR: ${{ secrets.REGRESS_TEST_RESULT_CONNSTR }}
 
+      - name: Pytest data format compatibility tests
+        uses: ./.github/actions/run-python-test-set
+        with:
+          build_type: ${{ matrix.build_type }}
+          test_selection: regress
+          needs_postgres_source: true
+          run_with_real_s3: true
+          real_s3_bucket: ci-tests-s3
+          real_s3_region: us-west-2
+          real_s3_access_key_id: "${{ secrets.AWS_ACCESS_KEY_ID_CI_TESTS_S3 }}"
+          real_s3_secret_access_key: "${{ secrets.AWS_SECRET_ACCESS_KEY_CI_TESTS_S3 }}"
+          rerun_flaky: true
+          extra_params: -m compatibility
+        env:
+          TEST_RESULT_CONNSTR: ${{ secrets.REGRESS_TEST_RESULT_CONNSTR }}
+
```

Comment:
This may not be the optimal way. Compatibility tests are long-running and should be run sequentially (at least 2 of them), i.e. on the same worker.

Maybe we can just `sed -i /-m 'not compatibility'// pytest.ini`, if pytest doesn't provide a better solution?

Reply:
`skipif` sounds good, let try it

Reply:
Done